src/huggingface_hub/keras_mixin.py

```diff
@@ -170,9 +172,9 @@ def save_pretrained_keras(
         plot_model (`bool`, *optional*, defaults to `True`):
             Setting this to `True` will plot the model and put it in the model
             card. Requires graphviz and pydot to be installed.
-        task_name (`str`, *optional*):
-            Name of the task the model was trained on. Available tasks
-            [here](https://github.com/huggingface/hub-docs/blob/main/js/src/lib/interfaces/Types.ts).
+        tags (`dict`, *optional*):
```

Comment:
The type here seems wrong, this should be a Union of `List` and `str` I think.

Reply:
I changed this.